packages/openid4vc/src/openid4vc-holder/OpenId4VciHolderService.ts

```diff
@@ -489,6 +490,8 @@ export class OpenId4VciHolderService {
         .withToken(tokenResponse.access_token)
 
       const credentialRequestClient = credentialRequestBuilder.build()
+      // retrive the context from the credential configuration
+      const context = offeredCredentialConfiguration.format === OpenId4VciCredentialFormatProfile.JwtVcJsonLd || offeredCredentialConfiguration.format === OpenId4VciCredentialFormatProfile.LdpVc ? (offeredCredentialConfiguration.credential_definition as CredentialDefinitionJwtVcJsonLdAndLdpVcV1_0_13)['@context']  : undefined
```

Comment:
I don't think the cast is needed

```suggestion
      // retrieve the @context from the credential configuration
      const context = offeredCredentialConfiguration.format === OpenId4VciCredentialFormatProfile.JwtVcJsonLd || offeredCredentialConfiguration.format === OpenId4VciCredentialFormatProfile.LdpVc ? offeredCredentialConfiguration.credential_definition['@context'] : undefined
```